web/views/FcRequestsTrack.vue

```diff
@@ -189,6 +212,15 @@ export default {
       });
       return itemsAll;
     },
+    numPages() {
+      return Math.ceil(this.total / this.itemsPerPage);
+    },
+    pageFrom() {
+      return (this.page - 1) * this.itemsPerPage + 1;
```

Comment:
minor bug: if `this.total === 0`, this will still return 1.  we should do `Math.min(0, ...)` here, or even just have an explicit `if (this.total === 0) { return 0; }` check.